ls1intum/Artemis
# `Development`: Add LTI 1.3 documentation

### Checklist
#### General



### Motivation and Context


Prepare a Github documentation page for instructors and admins to see what is working on LTI v1.3

### Description

Add instructions for lti steps specifically Moodle. 

### Steps for Testing

Use the following link to check the documentation: https://artemis-platform--7228.org.readthedocs.build/en/7228/user/lti.html
Please go through the documentation and check if it is understandable.

### Review Progress






#### Code Review

### Screenshots
